go.mod

```diff
@@ -35,6 +35,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/licensemanager v1.15.3
 	github.com/aws/aws-sdk-go-v2/service/sts v1.16.3
 	github.com/google/uuid v1.2.0
+	github.com/prometheus/client_model v0.2.0
```

Comment:
Would you need to run `go mod tidy` to update go.sum?

Reply:
No. This was a non-explicit import in the prior version so it was already pulled in.